README.md

```diff
@@ -27,22 +32,37 @@ Note the following images are deprecated and will only receive a last update for
 | nodejs       | A size compact GraalVM Community Edition container image with the [Node.js runtime](https://github.com/oracle/graaljs/blob/master/docs/user/NodeJS.md).                          | RPM-based<sup>1</sup> |
 | graalpy       | A size compact GraalVM Community Edition container image with the [GraalPy runtime](https://github.com/oracle/graalpython).                          | Standalone<sup>2</sup> |
 
-**1**: RPM-based GraalVM Community Edition container images are based on GraalVM components RPMs that are available for Oracle Linux 7, Oracle Linux 8 and Oracle Linux 9. Similar to any other available packages, you can install these components using `yum` on Oracle Linux 7 or `microdnf` on the Oracle Linux 8 and Oracle Linux 9 based images.
+**1**: RPM-based GraalVM Community Edition container images are based on GraalVM components RPMs that are available for Oracle Linux 7, Oracle Linux 8, and Oracle Linux 9. Similar to any other available packages, you can install these components using `yum` on Oracle Linux 7, or `microdnf` on Oracle Linux 8 and 9 based images.
+
+**2**: Standalone-based GraalVM Community Edition container images based on standalone builds of specific Truffle languages. These images are created as a drop-in replacement for other implementations, and do not contain a Java runtime.
+
+## Tags 
+
+There are multiple tags that let you choose the level of stability you need including the Java version, build number, and the Linux version. 
+Image tags use the following naming convention:
+
+```bash
+$version[-muslib(for native image only)][-$platform][-$buildnumber]
+```
 
-**2**: Standalone-based GraalVM Community Edition container images based on standalone builds of specific Truffle languages. These images are created as a drop-in replacement for other implementations (TruffleRuby instead of MRI Ruby), and do not contain a Java runtime.
+The following tags are listed from the most-specific tag (at the top) to the least-specific tag (at the bottom). 
+The most-specific tag is unique and always points to the same image, while the less-specific tags point to newer image variants over time.
 
-The images are tagged with the format `ghcr.io/graalvm/IMAGE_NAME:java_version`.
-The version tag defines the level of specificity.
-It is recommended that the most specific tag be used, for example, `20.0.1` or `20.0.1-ol9-date`, where the `` means the image required a patch and this specific build will never change.
```

Comment:
Please let's use tags for JDK21 as JDK20 is not supported anymore.

Reply:
We use JDK17 tags in the documentation so I copied.